test/bert/tokenizer/internal/tBasicTokenizer.m

```diff
@@ -1,7 +1,7 @@
 classdef tBasicTokenizer < matlab.unittest.TestCase
```

Comment:
You should add a test for the batched input case - something like `tokenize(tok,strs) == arrayfun(@(str) tokenize(tok,str),strs,UniformOutput=false)`

Reply:
It's not exactly equal, since tokenize will return one cell array for each call, but I have added a test to reflect that.